client/app/components/ui/required-asterisk.hbs

```diff
@@ -0,0 +1,6 @@
+<sup>
+  <FaIcon @icon="asterisk" class="text-red text-weight-normal" @transform="shrink-4" />
+  <EmberTooltip>
```

Comment:
What's the difference between this and `<LabsUi:Icontooltip>` ?
We should be consistent with use of components if possible

Reply:
! Good call. Thanks. I'll revise.